rust/src/daemon.rs

```diff
@@ -70,7 +70,13 @@ pub(crate) fn deployment_populate_variant(
 
     dict.insert("pinned", &deployment.is_pinned());
     let unlocked = deployment.get_unlocked();
-    dict.insert("unlocked", &unlocked.to_string());
+    // Unwrap safety: This always returns a value
```

Comment:
Ah and this would be fixed by https://github.com/ostreedev/ostree/commit/43913178a7eb23f615555dc0962844fce1f82570 after we update the rust bindings.